shap/explainers/_additive.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, model, masker):
             note that this structure information has no effect on the explanations of additive models.
         """
         super(Additive, self).__init__(model, masker)
+        self.name = 'additive explainer'
```

Comment:
can we add name to the base explainer and pass this through the init() method?

Reply:
same thing for other explainers below

Reply:
Sure. Will be taken care of in the next PR as well.